### PR TITLE
Pagination: convert Light DOM LESS to SCSS

### PR DESCRIPTION
**Pull request by BY00565233**
JIRA: SURF-1880
Converting the less to scss styles and removing the pagination less file.

### LGTM's
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/scss/components/_index.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 @import "badge/index";
 @import "breadcrumb/index";
 @import "icon/index";
+@import "pagination/index";
```
Comment (CITguy) [inline src/scss/components/_index.scss]: please alphabetize the imports

Comment (BY00565233) [reply src/scss/components/_index.scss]: Yes Ryan  done the changes and updated in git.

**Comment by CITguy** [comment]
@BY00565233: please squash to 1 commit

**Comment by BY00565233** [comment]
> @BY00565233: please squash to 1 commit

Yes Ryan done

**Pull request merged**